src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -765,6 +765,16 @@ public enum NuGetLogCode
         /// </summary>
         NU5126 = 5126,
 
+        /// <summary>
+        /// No ref or lib folder in the package
+        /// </summary>
+        NU5127 = 5127,
+
+        /// <summary>
+        /// No dependencies for each tfm in the package 
+        /// </summary>
+        NU5128 = 5128,
```

Comment:
Make sure you create a new git branch for each rule, and be careful to checkout the correct branch when changing what you're working on. `NU5128` is out of scope for the "no lib or ref" rule.